scripts/karma.conf.js

```diff
@@ -0,0 +1,33 @@
+const generate = require('videojs-generate-karma-config');
+
+module.exports = function(config) {
+  const options = {
+    customLaunchers(defaults) {
```

Comment:
I think the default videojs-generate-karma-config browser set up should work since contrib-eme doesn't have e2e playback tests